core/GlobalState.cc

```diff
@@ -1138,6 +1138,7 @@ unique_ptr<GlobalState> GlobalState::deepCopy(bool keepId) const {
     auto result = make_unique<GlobalState>(this->errorQueue);
 
     result->silenceErrors = this->silenceErrors;
+    result->runningUnderFuzzer = this->runningUnderFuzzer;
```

Comment:
there's already https://github.com/sorbet/sorbet/blob/master/common/common.h#L52 for this